.github/workflows/compareNDandODbuilds.yml

```diff
@@ -0,0 +1,107 @@
+name: Build and profile Android apps
+
+on:
+  workflow_dispatch:
```

Comment:
dont we need to specify PR number or some way to tell the workflow what code to build? `${{ github.event.pull_request.head.sha }}` - the event.pull_request is not defined in this case, is it?